src/Cake.Yaml/Cake.Yaml.csproj

```diff
@@ -1,7 +1,7 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFrameworks>netstandard2.0</TargetFrameworks>
+    <TargetFrameworks>net6.0;net5.0;netcoreapp3.1</TargetFrameworks>

```

Comment:
Isn't is still best practice to just target netstandard2.0 here, unless you need something from the other plattforms?

Reply:
[No, not anymore](https://cakebuild.net/docs/extending/addins/best-practices#net-target-version)... Cake v2.0.0 dropped `netstandard2.0` support for `Cake.Core` and `Cake.Common`.

https://cakebuild.net/blog/2021/11/cake-v2.0.0-released#net-6-and-c-10-support